libs/ui-lib/lib/ocm/components/clusterConfiguration/HostInventory.tsx

```diff
@@ -61,13 +57,6 @@ const HostInventory = ({ cluster }: { cluster: Cluster }) => {
           </TextContent>
         )}
       </StackItem>
-      {isPlatformIntegrationFeatureEnabled && (
-        <StackItem>
-          <Split>
-            <PlatformIntegration clusterId={cluster.id} />
```

Comment:
The components `PlatformIntegration` and `PlatformIntegrationVsphere`  are not used anymore.

Reply:
I've removed it